src/CursorPagination.php

```diff
@@ -194,6 +200,22 @@ public function withKeyName(string $column): self
         return $this;
     }
 
+    /**
+     * Set the decoder callback for paginator key column values.
+     *
+     * In case key column value is stored not as a string, for example uuid in binary format,
+     * it has to be decoded before passing it to Cursor.
+     *
+     * @param callable $callback
+     * @return $this
+     */
+    public function withKeyDecoder(callable $callback): self
+    {
+        $this->keyDecoderCallback = $callback;
```

Comment:
```suggestion
        $this->keyDecoderCallback = Closure::fromCallable($callback);
```
here or you may get a type error due to the Callable type on the property

Reply:
Done